examples/WebApp.AspNetCore/ApplicationInsightsDiagnostics.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "LogDirectory": ".",
+  "FileSize": 1024,
+  "LogLevel": "Verbose"
```

Comment:
Warning would be sufficient for most cases, unless investigating something very specific.